Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM elixir:1.3.4
+RUN mkdir -p /app
+WORKDIR /app
+CMD ["iex", "-S", "mix"]
+COPY . /app
+RUN mix local.hex --force && \
+    mix deps.get && \
+    mix compile && \
+    mix credo --strict
```

Comment:
Not a fan of having this be required by the build.  The output is valuable feedback, but failing a build on it will lead to frustration and bad code.